src/gfx/shaders/Volume_frag.glsl

```diff
@@ -39,16 +47,46 @@ vec4 sample3DTexture(vec3 texCoord)
   return mix(colorSlice1, colorSlice2, weightSlice2);
 }
 
+vec4 boxToTextCoord(vec3 boxCoord) {
+  /***************** YOZ *********************/
+  float currDeltaYZ = (1. - boxCoord.z) * deltaYZ;
+  if (PI / 2. - boxAngles.x > 0.) {
+    currDeltaYZ = boxCoord.z * deltaYZ;
+  }
+  if (currDeltaYZ > boxCoord.y  || boxCoord.y > 1. - deltaYZ + currDeltaYZ)
```

Comment:
Please try to rethink this entire function in vector arithmetic to perform similar calculations in parallel. Make use of vec3 form for deltas suggested in VolumeMesh.js, or even change these delta uniforms into something easier to use here. This function is called a lot of times in the shader, it should be blazing fast.

Reply:
I tried to vectorize and improve this part of the programme in a new commit.